src/odoc/bin/main.ml

```diff
@@ -424,12 +459,17 @@ end = struct
     let cmd =
       Term.(
         const handle_error
-        $ (const list_targets $ dst () $ back_compat $ R.extra_args $ input))
+        $ (const list_targets $ dst () $ back_compat $ R.extra_args
+         $ input_odocl))
 
     let info =
-      Term.info
-        (R.renderer.name ^ "-targets")
-        ~docs:section_support ~doc:"TODO: Fill in."
+      let doc =
+        Format.sprintf
+          "Print the files that would be generated by $(i,%s-generate). Works \
+           on both $(i,.odoc) and $(i,.odocl) files."
```

Comment:
While it's true that it works on `odoc` and `odocl` files, I don't think we want to advertise this. It should really only be run on `odocl` files.

Reply:
Right, let's remove this sentence.